components/raftstore/src/store/worker/region.rs

```diff
@@ -763,6 +817,14 @@ where
             }
             task @ Task::Apply { .. } => {
                 fail_point!("on_region_worker_apply", true, |_| {});
+                if self.ctx.coprocessor_host.should_pre_apply_snapshot() {
+                    if let Err(e) = self.ctx.pre_apply_snapshot(&task) {
+                        info!(
```

Comment:
maybe warn?

Reply:
I think maybe we can just remove this log. Since:
1. If the error is raised by check_abort, which means we just quit applying this snapshot, it is in expectation.
2. If the error is because we have not finished receiving/persisting snapshot for now, it is also in expectation.

For the rear case, we can have a `pre_apply_snapshot(.., None)` to notify, so we can print if we need in proxy code, though currently this is not necessary.